src/absil/illex.fsl

```diff
@@ -79,12 +80,7 @@ let kwdInstrTable =
 
 let kwdOrInstr s = (Lazy.force kwdInstrTable).[s] (* words *)
 
-let eval = function 
-  | '0' -> 0  | '1' -> 1 | '2' -> 2  | '3' -> 3  | '4' -> 4  | '5' -> 5 
-  | '6' -> 6  | '7' -> 7  | '8' -> 8  | '9' -> 9 
-  | 'A' -> 10 | 'B' -> 11 | 'C' -> 12 | 'D' -> 13 | 'E' -> 14 | 'F' -> 15
-  | 'a' -> 10 | 'b' -> 11 | 'c' -> 12 | 'd' -> 13 | 'e' -> 14 | 'f' -> 15
-  | _ -> failwith "bad hexbyte"  
+let evalDigit ch  = (int ch) - (int '0')
```

Comment:
Good change - was only called within the range '0' - '9'. Name makes more sense too.

Reply:
consider to write  `- 48`!?

Reply:
Actually the compiler is smart enough infer that it is "-48" so I will keep the existing code since I think the intent is more clear than to use "-48".

Reply:
good. just wanted to know if this is already optimized away ;-)